apps/vs-code-designer/src/app/commands/generateDeploymentScripts/generateDeploymentScripts.ts

```diff
@@ -52,7 +52,7 @@ export async function generateDeploymentScripts(context: IActionContext, node?:
     ext.outputChannel.appendLog(localize('initScriptGen', 'Initiating script generation...'));
 
     addLocalFuncTelemetry(context);
-    if (node) {
+    if (Object.keys(node).length > 0 && (await isLogicAppProject(node.fsPath))) {
```

Comment:
Can we please check that node is not undefined or null. I can see that the `node` in the params is optional, this means that is can be undefined (thats why we are checking it here). Doing `Object.keys(undefined)` will throw an error